transport/EndpointMetrics.h

```diff
@@ -0,0 +1,33 @@
+#include <cstdint>
+
+#pragma once
+
+struct EndpointMetrics
+{
+    EndpointMetrics(uint32_t sQueue, double rKbps, double sKbs)
+        : sendQueue(sQueue)
+        , receiveKbps(rKbps)
+        , sendKbps(sKbs)
+    { }
+
+    EndpointMetrics() : EndpointMetrics(0, 0.0, 0.0) { }
+
+    EndpointMetrics& operator+=(const EndpointMetrics& rhs)
+    {
+        sendQueue += rhs.sendQueue;
+        receiveKbps += rhs.receiveKbps;
+        sendKbps += rhs.sendKbps;
+        return *this;
+    }
+
+    uint32_t sendQueue;
+    double receiveKbps;
```

Comment:
I wonder if it is better to store in kbps. No need for the precision I think.

Reply:
Maybe you were seeing my old branch where I store in bits/s? It's stored in kbps since I add this change to this branch. Or am I getting something wrong?